ConnectorAutocadCivil/ConnectorAutocadCivil/Utils.cs

```diff
@@ -354,9 +452,9 @@ public static void SetPropertySets(this Entity entity, Document doc, List<Dictio
     {
       propertySets = PropertyDataServices.GetPropertySets(obj);
     }
-    catch (Exception e)
-    { }
-    if (propertySets == null)
+    catch (Autodesk.AutoCAD.Runtime.Exception) { }
```

Comment:
What's the rationale for continuing? if continung is valid, we should add the rationale in the {}

Reply:
Continuing because this may throw if the object simply doesn't have property sets - there is no way to check for property sets before attempting to retrieve them on an object with `GetPropertySets` afaik. adding it into comments